### PR TITLE
1060 UI talent details page

### LINKED ISSUE #1060: UI changes for Talent details page
UI changes for Talent details page in catalog
Use https://app.zeplin.io/project/62b9dba49dd2b211410e608e/screen/62b9dbf684718712f34fe8d0 as reference

### PR DESCRIPTION
**Pull request by minump**
## Description
Updated styling of talent details page (similar to capability details page).
Changed col space and styling.
See catalog/contributions/contributionID/talents/talentID page

**Fixes #1060**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/templates/contribute/talent_details.html [hunk 1]
```diff
@@ -20,208 +21,221 @@
...
+        {% if post.requiredBuildingBlocks %}
         <div class="row">
-            <div class="col-12 col-sm-2">
-                <h4>Start Date</h4>
+            <div class="col-12 col-md-3">
+                <h4>Required Building Blocks</h4>
             </div>
-            <p class="col-sm-9">{{post.startDate}}</p>
+            <table>
+                {% for x in post.requiredBuildingBlocks %}
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/templates/contribute/talent_details.html:95]: The Required Building Blocks section alignment looks off. I thought of sharing as feedback as it's related to the user interface. Thanks.

 
image

The rest of the changes look fine to me. Thanks.

Comment (minump) [reply contributions/catalog/webapps/templates/contribute/talent_details.html:98]: Added a column div to fix styling.

Ready for review.

Comment (sandeep-ps) [reply contributions/catalog/webapps/templates/contribute/talent_details.html:100]: Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
Hi, @bingzhang; I think this PR may be ready for merging. If you won't be able to review it soon, please let Minu know if it's okay to merge without your approval. Thanks.

**Comment by minump** [comment]
Merging this PR.

**Pull request merged**